knative-operator/pkg/common/constants.go

```diff
@@ -6,4 +6,6 @@ const (
 	ServingOwnerNamespace            = "serving.knative.openshift.io/ownerNamespace"
 	ServerlessOperatorOwnerName      = "operator.knative.openshift.io/ownerName"
 	ServerlessOperatorOwnerNamespace = "operator.knative.openshift.io/ownerNamespace"
+	KafkaOwnerName                   = "kafka.knative.openshift.io/ownerName"
+	KafkaOwnerNamespace              = "kafka.knative.openshift.io/ownerNamespace"
```

Comment:
Why this group? I'd think this should be `knativekafkas.operator.serverless.openshift.io` to play nice with the group used for the API.

Reply:
+1

Reply:
Done